### DISCUSSION TITLE
Contact Us form

### DISCUSSION THREAD

**Discussion by leachuk** [ldenoue/readabletranscripts#1] [Ideas]
You might be interested in adding a Contact Us form to your site. If you are, may I cheekily point you to a free thing I made to help https://first-contact.net

You can read about it here https://news.ycombinator.com/item?id=42277763

Good luck with your launch.

**Comment by ldenoue** [comment]
As a quick feedback I recommend you implement some sort of social login (I like Google for example) instead for your Sign Up. You probably will get more people trying your tool.

**Reply by leachuk** [reply]
Yes, this is good advice and it's on my list. Thanks :-)
